File: Angular/Hahn_Softwareentwicklung/src/app/models/shippingAddress.model.ts
export class ShippingAddressModel {
    id: string;
    addressLine1: string;
    addressLine2: string;
    city: string;
    state: string;
    postalcode: string;
    country: number;

    constructor(id: string, addressLine1: string, addressLine2: string, city: string, state: string, postalcode: string, country: number) {
        this.id = id;
        this.addressLine1 = addressLine1;
        this.addressLine2 = addressLine2;
        this.city = city;
        this.state = state;
        this.postalcode = postalcode;
        this.country = country;
    }
  }

File: Backend/Models/ShippingAddressModel.cs
namespace Hahn_Softwareentwicklung.Models
{
    public class ShippingAddressModel
    {
        public Guid Id { get; set; }
        public string AddressLine1 { get; set; }
        public string AddressLine2 { get; set; }
        public string City { get; set; }
        public string State { get; set; }
        public string PostalCode { get; set; }
        public string Country { get; set; }

        public ShippingAddressModel(Guid id, string addressLine1, string addressLine2, string city, string state, string postalCode, string country)
        {
            Id = id;
            AddressLine1 = addressLine1;
            AddressLine2 = addressLine2;
            City = city;
            State = state;
            PostalCode = postalCode;
            Country = country;
        }
    }
}
